spec/models/waste_carriers_engine/registration_spec.rb

```diff
@@ -566,6 +566,40 @@ module WasteCarriersEngine
       end
     end
 
+    describe "#pending_payment?" do
```

Comment:
if we `alias`, we won't need to test this, instead we should test `unpaid_balance?`. Also, this test is not a pure unit, shall we still go for it?